src/Components/FormCreation/Submission/SubmissionFormStateManagement.ts

```diff
@@ -6,77 +6,6 @@ import {
   FormSectionBase
 } from "../../../Types/FormTypes";
 
-export const sectionsStarter: FormSectionBase[] = [
-  {
-    name: "About Your Charity",
```

Comment:
Removed consts already exported from CreateEditFormStateManagement. 

We can modularize this to a common folder later.